charts/portworx/values.yaml

```diff
@@ -3,9 +3,7 @@
 deployOperator: true                    # Deploy the Portworx operator
 deployCluster: true                     # Deploy the Portworx cluster
 
-deploymentType: oci                     # accepts "oci" or "docker"
-imageType: none                         #
-imageVersion: 2.9.1.4                   # Version of the PX Image.
+imageVersion: 2.11.0                    # Version of the PX Image.
```

Comment:
We still default to 2.9.1.4 from spec generator, let's stick to that. 
https://install.portworx.com/version